Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/CounterFactory.cs

```diff
@@ -0,0 +1,615 @@
+                        reportAs,
+                        "largeObjectHeapSize",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\Finalization Survivors":
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# GC Handles":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "gcHandles",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\Allocated Bytes/ sec":
+                    return new RateCounterGauge(
+                        reportAs,
+                        "allocatedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Induced GC":
+                    return new RateCounterGauge(
+                        reportAs,
+                        "inducedGC",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Bytes in all Heaps":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "bytesInAllHeaps",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Total committed Bytes":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "commitedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# Total reserved Bytes":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "reservedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\.NET CLR Memory(??APP_CLR_PROC??)\# of Pinned Objects":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "pinnedObjects",
+                        AzureWebApEnvironmentVariables.CLR);
+                
+                //// Quick pulse related hard coded performance counters.
+                case @"\Memory\Committed Bytes":
+                    return new RawCounterGauge(
+                        reportAs,
+                        "committedBytes",
+                        AzureWebApEnvironmentVariables.CLR);
+                case @"\Processor(_Total)\% Processor Time":
+                    return new CPUPercenageGauge(
+                        reportAs, 
+                        new SumUpCountersGauge(
+                            reportAs,
+                            new RawCounterGauge("kernelTime", "kernelTime", AzureWebApEnvironmentVariables.App),
+                            new RawCounterGauge("userTime", "userTime", AzureWebApEnvironmentVariables.App)));
+                case @"\ASP.NET Applications(__Total__)\Requests In Application Queue":
```

Comment:
Same here. Total requests in queue is not the same as the current process requests in queue.